viewer.jsx

```diff
@@ -1,19 +1,23 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import GeoNodeViewerDebug from './GeonodeDebug.jsx';
+import GeoNodeViewer from './geonode.jsx';
 import enMessages from 'boundless-sdk/locale/en.js';
 import {IntlProvider} from 'react-intl';
 
 class Viewer {
   constructor(domId, config) {
     this.domId = domId;
     this.mapConfig = config;
+    this.proxy = undefined;
   }
   set config(value) {
-    this.mapConfig = config;
+    this.mapConfig = value;
+  }
+  set setProxy(value) {
```

Comment:
should both setters have the same signature? i.e. if this is called setProxy, should the other one be called setMapConfig instead?